hitl/run.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
@jordens I had to revert back to using `bash` here instead of `sh`. We use the `source` command` below, which is explicit to `bash` and is unavailable in `sh`.

Reply:
`source` is a bashism. https://wiki.ubuntu.com/DashAsBinSh#source